rllib/policy/policy_map.py

```diff
@@ -34,29 +24,25 @@ class PolicyMap(dict):
 
     def __init__(
         self,
-        worker_index: int,
-        num_workers: int,
+        *,
```

Comment:
Simplified PolicyMap a lot, making it a very thin wrapper around a dict and removing the "heavy" add_policy/create_policy APIs. Users should create policies outside the PolicyMap and just use
```
policy_obj = ctor(a=.., b=.., c=..)
policy_map[some_id] = policy_obj
```
instead.

Reply:
Awesome!